src/RandomHelper.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+abstract contract RandomHelper {
+    /// @notice Shuffle an array using Fisher-Yates algorithm
+    /// @dev Based on https://gist.github.com/scammi/602387a22e04c77beb73c0ebc0f0bc18
+    function shuffleArray(uint256[] memory shuffle, uint256 entropy) internal {
```

Comment:
Need to return the array, since it's just being shuffled in memory

Reply:
No, this actually works. I'm using it in one codebase :slightly_smiling_face:

Reply:
Alright sounds good 🙂